src/components/PageHero.js

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import { StaticImage } from "gatsby-plugin-image"
+
+class PageHero extends React.Component {
+    render() {
+        const {title, image, subTitle} = this.props
+        return (
+            <div className="page-hero">
+                <StaticImage 
+                        src={ image } 
+                        alt={ title } 
+                        className="page-hero_img"
+                        layout="fixed"
+                />
```

Comment:
Static image won't work with passed references per the documentation. For now just use something like:

```suggestion
                <img
                        src={image}
                        alt={title}
                        className="page-hero_img"
                />
```

Later these should be swapped for the gatsby image component